WalletWasabi/Services/UpdateManager.cs

```diff
@@ -28,7 +28,7 @@ public UpdateManager(TimeSpan period, string dataDir, bool downloadNewVersion, I
 		GithubHttpClient = githubHttpClient;
 		WasabiClient = sharedWasabiClient;
 		// The feature is disabled on linux at the moment because we install Wasabi Wallet as a Debian package.
-		DownloadNewVersion = downloadNewVersion && !RuntimeInformation.IsOSPlatform(OSPlatform.Linux);
+		DownloadNewVersion = downloadNewVersion && (RuntimeInformation.IsOSPlatform(OSPlatform.OSX) || RuntimeInformation.IsOSPlatform(OSPlatform.Windows));
```

Comment:
Here I changed slightly because it better fits with the rest of the logic in `GetAssetToDownload`